packages/plans-grid-next/src/hooks/data-store/use-grid-plans.tsx

```diff
@@ -259,7 +259,7 @@ export const usePlanTypesWithIntent = ( {
 			planTypes = [ TYPE_FREE, TYPE_PERSONAL, TYPE_PREMIUM, TYPE_BUSINESS, TYPE_ECOMMERCE ];
 			break;
 		case 'plans-playground':
-			planTypes = [ TYPE_BUSINESS, TYPE_ECOMMERCE ];
+			planTypes = [ TYPE_FREE, TYPE_PERSONAL, TYPE_PREMIUM, TYPE_BUSINESS, TYPE_ECOMMERCE ];
```

Comment:
`TYPE_BLOGGER` is also a plan which we might want to include but I don't know much about it.

Reply:
The blogger plan has been retired and is no longer available for purchase. It remains in the other part of the code for retrocompatibility.

Reply:
But does that not mean, a simple site on blogger plan can also attempt to apply a blueprint on itself some day?